portal/import_template_ui.php

```diff
@@ -701,7 +701,11 @@ function popGroupsDialog() {
                                 }
                                 $next_due = $templateService->showTemplateFromEvent($file, true);
                                 if ($next_due > 1) {
-                                    $audit_status['denial_reason'] = xl('Scheduled');
+                                    if ($audit_status['denial_reason'] === 'In Review') {
+                                        $audit_status['denial_reason'] = xl('Scheduled') . ' ' . xl('but Needs Review');
```

Comment:
this seems silly could have been
 $audit_status['denial_reason'] = xl('Scheduled but Needs Review.')

oh well, hindsight...